### PR TITLE
Improve HTML in request show redesign's conversation tab

### LINKED ISSUE #12937: Layout issues if a request is in accepted state
See https://build.opensuse.org/request/show/993778

### PR DESCRIPTION
**Pull request by dmarcoux**
This made the HTML somewhat hard to follow and caused issues in the layout when the request description is a collapsible text element.

Fixes #12937

Before, with the issue highlighted in the HTML:
issue

After:
after

### CODE REVIEW

**Comment by dmarcoux** [comment]
[Comments and request history is too narrow](https://trello.com/c/A2EgWAux/1951-comments-and-request-history-is-too-narrow)

**Comment by hennevogel** [comment]
Are reviews now below the comment box again?

**Review by hennevogel — APPROVED** [review]

**Comment by dmarcoux** [comment]
> Are reviews now below the comment box again on smaller viewports?

They are below the request description, but above the comments & request history:
preview
